CodeLite/winprocess.cpp

```diff
@@ -173,26 +173,22 @@ WinProcess::~WinProcess() {}
 
 bool WinProcess::Read(wxString& buff)
 {
-    DWORD dwRead;
-    DWORD dwMode;
-    DWORD dwTimeout;
-    char* chBuf = new char[65536 + 1]; // 64K should be sufficient buffer
-    memset(chBuf, 0, 65536 + 1);
-
-    std::unique_ptr<char> sp(chBuf);
```

Comment:
wouldn't be sufficient just to change this into: 

```c++
std::unique_ptr<char[]> sp(chBuf);
```

Reply:
It was possible too. `std::vector` seems simpler/more appropriate IMO.